public/robots.xml

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: /
+
+Sitemap: https://chained-assets.vercel.app/sitemap.xml
```

Comment:
Update url? `https://chained-assets.vercel.app/sitemap.xml`

Reply:
yes. it should be updated